.circleci/config.yml

```diff
@@ -35,16 +35,16 @@ jobs:
       - run: python setup.py egg_info
 
   html-docs:
-    environment:
-      - MAIN_CMD: "venv/bin/python setup.py"
-      - SETUP_CMD: "build_docs -w"
     docker:
       - image: circleci/python:3.6
     steps:
+      - restore_cache:
+          keys: sample-data-v1
+
```

Comment:
Why this blank line?

Reply:
just to make it a little easier to separate the cache restore from the build part when reading the yaml.